x-pack/plugin/esql/qa/testFixtures/src/main/resources/floats.csv-spec

```diff
@@ -1,4 +1,11 @@
 // Floating point types-specific tests
+parseLargeMagnitudeValues
+required_capability: fix_parsing_large_negative_numbers
+row a = 92233720368547758090, b = -92233720368547758090;
```

Comment:
I'd also add the previously failing example from the bug report, just for completion (I also like the use of `-9223372036854775809` as first integer below Long.MIN_LONG that should be turned into a double).